Sofa/Component/Topology/Mapping/src/sofa/component/topology/mapping/Edge2QuadTopologicalMapping.cpp

```diff
@@ -97,6 +97,51 @@ void Edge2QuadTopologicalMapping::init()
     }
 
     unsigned int N = d_nbPointsOnEachCircle.getValue();
+    bool modelsOk = true;
+
+    // Check input topology
+    if (!fromModel)
+    {
+        // If the input topology link isn't set by the user, the TopologicalMapping::create method tries to find it.
+        // If it is null at this point, it means no input mesh topology could be found.
+        msg_error() << "No input mesh topology found. Consider setting the '" << fromModel.getName() << "' data attribute.";
+        modelsOk = false;
+    }
```

Comment:
2 remarks regarding this check:
- This case is never possible because fromModel is already checked in the mother class TopologicalMapping in the canCreate method. If input and output don't link to valid BaseMeshTopology, the mapping is not create (canCreate return false)
- Hopefully this case is never possible because if fromModel is null, printing error message with fromModel.getName() will crash 😄